pageserve/templates/events.html

```diff
@@ -35,8 +32,6 @@ <h1>{% block title %}Sub-Events{% endblock title %}</h1>
 <hr>
 {% endfor %}
 
-<a href='/v1/'>See all posts</a>
```

Comment:
What happened to these links? I don't see them in `base.html` either.

Reply:
I actually moved them into the navbar, so there's no more need for links to other pages within the page body.